examples/vue-todos/.vscode/components.code-snippets

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Are you sure? I find this to be quite helpful for others as it points to the suggested vs-code plugins that are recommended and in the VueJS community this is somewhat more important than normal because there are three different plugin's which can handle the typing for `.vue` files and these files helps to disambiguate which plugins are intended to be used.

Reply:
ok that last comment was for the `extensions.json` ... this one is just a helper to allow a user to add a `.vue` file more quickly. I'll remove it if you like but in general I think it's more helpful than hurtful. I'd be interested in why you would want to remove this.

Reply:
Well it's just a general rule to not commit editor specific files, like .idea/ and .vscode/, but if this isn't the case in the vue community i let someone else be the judge here.